### DISCUSSION TITLE
Changelog

### DISCUSSION THREAD

**Discussion by nimish** [marlonrichert/zsh-snap#145] [Ideas]
znap moves quickly and a changelog would be very helpful to understand best practices/updates to how to configure it properly

**Comment by marlonrichert** [comment]
What best practices do you need that are not already in the Readme and the [example `.zshrc` file](https://github.com/marlonrichert/zsh-snap/blob/main/.zshrc)?

**Reply by marlonrichert** [reply]
@nimish ?

**Reply by nimish** [reply]
Mostly a summary of what has changed without having to mine the git logs

**Comment by MartinDelille** [comment]
I'm watching the project which generate a lot of notification. I would enjoy if there was some release so I could get notified only for those.

**Reply by marlonrichert** [reply]
I didn't even know GitHub had such a feature. Thanks for the tip! I'll look into it.

**Reply by marlonrichert** [reply]
There, here's the first release: https://github.com/marlonrichert/zsh-snap/releases

I will make a new release whenever the code has been stable for a while.

**Reply by MartinDelille** [reply]
I missed your answer! Thanks! I will be able to tune the notification with greater precision now! 😄
